tests/functional/time/testcase.yaml

```diff
@@ -1,8 +1,8 @@
 tests:
   sidewalk.functional.time:
     tags: Sidewalk
-    platform_allow: nrf52840dk/nrf52840 nrf5340dk/nrf5340/cpuapp nrf54l15pdk@0.3.0/nrf54l15/cpuapp
+    platform_allow: nrf52840dk/nrf52840 nrf5340dk/nrf5340/cpuapp
     integration_platforms:
       - nrf52840dk/nrf52840
       - nrf5340dk/nrf5340/cpuapp
-      - nrf54l15pdk@0.3.0/nrf54l15/cpuapp
+      # - nrf54l15pdk@0.3.0/nrf54l15/cpuapp issue KRKNWK-18982
```

Comment:
Do not left commented code - I think the commit descrition is good enough, no need to repeat it here

Reply:
removed the line entirely. My initial intention was to leave in the code the reason why this test is not enabled, as the commit message is not convenient to check out while investigating the source code. 
But as we are already planning to take care of this issue, it is not a big deal to just delete it and do not leave any commented configuration